connector_clickup/security/ir.model.access.csv

```diff
@@ -0,0 +1,6 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
Only connector manager should perform the CRUD operations on clickup records, non-connector user only can read the data